ui/page/proposal/page.go

```diff
@@ -0,0 +1,200 @@
+package proposal
```

Comment:
It's misleading naming this file as `Page` because this is not the only page in this package, I can't tell if this is for the proposal details or list without looking into the code, can keep the name as "Proposals".